app/controllers/course/forum/posts_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Course::Forum::PostsController < Course::Forum::Controller
+  include Course::Discussion::PostsConcern
+
```

Comment:
Putting it in the topic retrieval method is not optimal. You can come up with an ordering that makes sense.